### PR TITLE
Adapt components to new data format

### PR DESCRIPTION
**Pull request by Hakimovich99**
Also fixed pipeline.py and load 1000 rows instead because questions (in evaluation pipeline) were generated from it 

NB: 
- to test pipeline_eval.py, must put full path as argument for custom components and use embed_text as a generic component (where fondant 0.8.dev2 is specified in component specs)
- still a draft since uses new data format and pipeline_eval might need to be updated as well. tbd.

### CODE REVIEW

**Comment by Hakimovich99** [comment]
> Also fixed pipeline.py and load 1000 rows instead because questions (in evaluation pipeline) were generated from it
> 
> NB:
> 
> * to test pipeline_eval.py, must put full path as argument for custom components and use embed_text as a generic component (where fondant 0.8.dev2 is specified in component specs)
> * still a draft since uses new data format and pipeline_eval might need to be updated as well. tbd.

**"and use embed_text as a generic component (where fondant 0.8.dev2 is specified in component specs)"** --> **fixed**

File: src/.fondant/compose.yaml
[the remark this answers is not in the document]
Comment (Hakimovich99) [reply src/.fondant/compose.yaml:1]: Sure!

File: src/components/retriever_eval/fondant_component.yaml [hunk 1]
```diff
@@ -5,27 +5,23 @@ image: ghcr.io/ml6team/retriever_eval:dev
...
-        type: string
-      contexts:
-        type: array
-        items:
-          type: string
-      context+precision:
-        type: float32
-      context+relevancy:
-        type: float32
+  question: #TODO: keep question and contexts, add/retrieve chosen metrics to compute
```
[the remark this answers is not in the document]
Comment (Hakimovich99) [reply src/components/retriever_eval/fondant_component.yaml:15]: Alright!

**Review by RobbeSneyders — APPROVED** [review]
Thanks @Hakimovich99!

**Pull request merged**